extensions/objects/templates/algebraic-expression-editor.component.spec.ts

```diff
@@ -97,7 +97,7 @@ describe('AlgebraicExpressionEditor', () => {
     fixture = TestBed.createComponent(
       AlgebraicExpressionEditorComponent);
     component = fixture.componentInstance;
-    windowRef.nativeWindow.Guppy = MockGuppy;
+    windowRef.nativeWindow.Guppy = MockGuppy as unknown as Guppy;
```

Comment:
Again, TODO missing

Reply:
Hmm, I get the point with the partial object, I think it should still be explained with a comment, mainly because we want to lower the chance of copying this pattern to places where it should not be used.

Reply:
Done!